test/core/test_searchhandler.cc

```diff
@@ -56,6 +57,7 @@ const std::map<TestCol, std::pair<std::string, std::string>> testColMap = {
     { TestCol::Id, { "t", "id" } },
     { TestCol::ItemId, { "t", "item_id" } },
     { TestCol::PropertyName, { "t", "property_name" } },
+    { TestCol::PropertyName2, { "t", "property_name2" } },
```

Comment:
Shouldnt this be `TestCol::PropertyName` again?

Reply:
I simulated `part_number` and `track_number`